parm/config/gfs/config.anal

```diff
@@ -19,12 +19,8 @@ if [[ "${CDUMP}" = "gfs" ]] ; then
   export DIAG_TARBALL="YES"
 fi
 
-export npe_gsi=${npe_anal}
-
-if [[ "${CDUMP}" == "gfs" ]] ; then
-  export npe_gsi=${npe_anal_gfs}
-  export nth_anal=${nth_anal_gfs}
-fi
+npe_var="npe_anal_${RUN//enkf}"
+export npe_gsi="${!npe_var}"
```

Comment:
What happened to `nth_anal`?

Reply:
Since `nth_anal` and `nth_anal_gfs` were the same, I consolidated them to just `nth_anal` in `config.resources`.